app/services/arcgis_api/geocoder.rb

```diff
@@ -124,6 +140,7 @@ def parse_address_candidates(response_body)
           city: candidate.dig('attributes', 'City'),
           state: candidate.dig('attributes', 'RegionAbbr'),
           zip_code: candidate.dig('attributes', 'Postal'),
+          addr_type: candidate.dig('attributes', 'Addr_type')
```

Comment:
are we using attr_type anywhere yet? are there common enum values we could document anywhere?